keps/NNNN-kep-template/README.md

```diff
@@ -155,7 +155,9 @@ notes and implementing the feature itself.  KEP editors, SIG Docs, and SIG PM
 should help to ensure that the tone and content of the `Summary` section is
 useful for a wide audience.
 
-A good summary is probably at least a paragraph in length.
+A good summary is probably at least a paragraph in length. Wrap lines (both
+in this section and elsewhere) to a reasonable length to make it easier for
+reviewers to cite specific portions.
```

Comment:
... and to minimize the diff when updating just a few lines in a larger paragraph.

:+1: